tests/app/helpers.py

```diff
@@ -86,6 +86,20 @@
     "SQE2a": ["as a prime contractor, using third parties (subcontractors) to provide some services"]
 }
 
+EMPTY_G7_DRAFT = {
```

Comment:
I'd make this a function that returns a new dictionary each time it's called (or even better, a py.test fixture). That gets rid of all the `.copy()` calls and makes sure that it or any of the nested items are never accidentally modified.